### DISCUSSION TITLE
TInvoicePaymentMeansCode "ipmc_CreditCard"

### DISCUSSION THREAD

**Discussion by dsshad** [LandrixSoftware/XRechnung-for-Delphi#15] [Q&A]
Der Code "ipmc_CreditCard" ist ja in TInvoicePaymentMeansCode enthalten. Die dazugehörige Gruppe und Elemente in den Spezifikationen müssten (BG-18) bzw. (BT-87 und BT-88) sein.

Wird der Code aktuell nicht weiter ausgewertet / verwendet, oder übersehe ich etwas?

Ist vermutlich ein eher seltener Fall hier, aber so für's Verständnis.

**Comment by landrix** [comment]
Muss ich mir anschauen

https://projekte.kosit.org/xrechnung/xrechnung/-/issues/76

https://www.arbeitsagentur.de/datei/muster-e-rechnung-inklusive-feldinhalte_ba030920.pdf
